### DISCUSSION TITLE
any significant competitors?

### DISCUSSION THREAD

**Discussion by ykdojo** [ykdojo/defaang#96] [General]
I started this project because I thought it might be something that'll be useful to a lot of people based on the current market conditions:

1. High salaries at FAANG+
2. Hard interviews at these companies
3. Lack of similar resources

For 3, I know that there are already other sites that do this, but I thought we can potentially make something much better.

Do you agree or disagree? Any thoughts on this? I'm curious to see what y'all think 😊

**Comment by kaleem68** [comment]
I like the initiative for defaang. I think it can serve more than what it promises ("A website that will curate recently-asked interview questions from FAANG+ to help people practice & prep!")

It can serve to:
 - People who would like to contribute to OS are preparing for tech interviews.
   Practicing leetcode in isolation or by own is hard and often bad for mental health so this is will be great community support.
